fhir-search/src/test/java/com/ibm/fhir/search/parameters/MultiTenantSearchParameterTest.java

```diff
@@ -53,7 +53,7 @@ public void testGetApplicableSearchParameters2() throws Exception {
         List<SearchParameter> result = SearchUtil.getApplicableSearchParameters("Medication");
         assertNotNull(result);
         printSearchParameters("testGetApplicableSearchParameters2", result);
-        assertEquals(23, result.size());
+        assertEquals(23 * 2, result.size());
```

Comment:
`* 2, ` has a particular purpose.  Can we throw this in a Constant that explains it? 
SearchTest.NUMBER_2_DOES_X?

Reply:
Because we're now indexin the list by both the code and the canonical uri, the expected size is now 2 times the number of search parameters.
I don't even love these tests which just happen to know the number of expected parameters in the file...seems like it would be better to just compute it from the file...otherwise its for sure gonna break each time we add or remove a search parameter.